xds/src/main/java/io/grpc/xds/internal/sds/SslContextProvider.java

```diff
@@ -33,7 +32,6 @@
  * stream that is receiving the requested secret(s) or it could represent file-system based
  * secret(s) that are dynamic.
  */
-@Internal
 public abstract class SslContextProvider<K> {
```

Comment:
This too doesn't need to be public any more?

Reply:
It is used by `io.grpc.xds.CdsLoadBalancer`.